Data/Classes/ProfessionGear.lua

```diff
@@ -34,13 +34,20 @@ function CraftSim.ProfessionGear:SetItem(itemLink)
 	end
 
 	self.item = Item:CreateFromItemLink(itemLink)
+	itemLink = self.item:GetItemLink()
 
 	-- parse stats
 	local extractedStats = GetItemStats(itemLink)
 
 	if not extractedStats then
 		print("Could not extract item stats: " .. tostring(itemLink))
-		return
+		itemLink = select(2, C_Item.GetItemInfoInstant(itemLink))

```

Comment:
why do we need a second try here?

Reply:
No, i've tested again and it did not change anything. It's a test initially